### PR TITLE
Upload Assignment 7 and 8

### PR DESCRIPTION
**Pull request by Prashant698**

### CODE REVIEW

File: prashant/HTML/Assignment7/professional.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    background-color: rgba(0,0,0,0.5) !important;
+    max-width: 600px;
+    margin: auto;
+    padding: 50px 70px;
+    border-top-left-radius: 30px;
+    border-top-right-radius: 30px;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.075);
+
+}*/
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/professional.css]: remove commented code

File: prashant/HTML/Assignment7/Login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+						</div>
+						<div class="d-flex justify-content-center">
+							<a href="#" style="color:#2d66fc ;">Forgot your password?</a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</body>
+</html>
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/Login.html]: add extra line after `</html>` .that is at the end.

File: prashant/HTML/Assignment7/homepage.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </header>   
+        <h1 style="text-align: center; margin-top: 150px; font-size: 50px;">Find The Best Job For Your Future</h1>
+        <div class="form-group">
+            <div style="margin-left: 300px; margin-top: 150px; ">
+                <input type="text" placeholder="Search Job Here" class="form-control" id="usr" style="width: 800px; ; height: 50px; font-size: 20px; padding-left: 20px;">
+                <button type="button" class="btn btn-primary" style="height: 60px; width: 200px; margin-left: -10px; background-color: blue;" >Search</button>
+            </div>
+        </div>     
+    </body>
+</html>
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/homepage.html]: add extra line after `</html> `.that is at the end.

File: prashant/HTML/Assignment7/login.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+.card-header h3 { 
+color: white;
+}
+
+.social_icon{
+position: absolute;
+right: 20px;
+top: -45px;
+}
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/login.css]: remove space

File: prashant/HTML/Assignment7/login.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+font-size: 60px;
+margin-left: 10px;
+color: #FFC312;
+}
+
+.social_icon span:hover {
+color: white;
+cursor: pointer;
+}
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/login.css]: remove space

File: prashant/HTML/Assignment7/personaldetail.css [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+}
+
+/*.card {
+    height: 270px;
+    margin-top: auto;
+    margin-bottom: auto;
+    width: 400px;
+    background-color: rgba(0,0,0,0.5) !important;
+    }*/
+
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/personaldetail.css]: remove commented code

File: prashant/HTML/Assignment7/personaldetail.css [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+/*.acedmic-form {
+    background-color: rgba(0,0,0,0.5) !important;
+    max-width: 600px;
+    margin: auto;
+    padding: 50px 70px;
+    border-top-left-radius: 30px;
+    border-top-right-radius: 30px;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.075);
+
+}*/
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment7/personaldetail.css]: remove commented code

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]
do suggested changes

File: prashant/HTML/Assignment7/professional.css (hunk 1 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/professional.css]: done

File: prashant/HTML/Assignment7/homepage.html (hunk 3 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/homepage.html]: done

File: prashant/HTML/Assignment7/login.css (hunk 5 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/login.css]: done

File: prashant/HTML/Assignment7/personaldetail.css (hunk 6 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/personaldetail.css]: done

File: prashant/HTML/Assignment7/personaldetail.css (hunk 7 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/personaldetail.css]: done

**Review by sourabh-kanwade — APPROVED** [review]

File: prashant/HTML/Assignment7/Login.html [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+		<title>Login Page</title>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+		<link rel="stylesheet" type="text/css" href="login.css">
+	</head>
+	<body>
+		<header>
+			<div class="container">
+				<div class="fixed-top">
+					<div class="header"></div>
```
Comment (dayanandviveki) [inline prashant/HTML/Assignment7/Login.html:17]: If this div not need then Remove it

File: prashant/HTML/Assignment7/homepage.css [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -ms-flex-align: center;
+    align-items: center;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    padding: 5px;
+}
+
```
Comment (dayanandviveki) [inline prashant/HTML/Assignment7/homepage.css]: Remove extra lines

File: prashant/HTML/Assignment7/Login.html (hunk 8 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/Login.html:17]: use ha uska

File: prashant/HTML/Assignment7/login.css [hunk 10]
```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css?family=Numans');
+
+html,body { 
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100%;
+    font-family: 'Numans', sans-serif;
+}
+
```
Comment (dayanandviveki) [inline prashant/HTML/Assignment7/login.css]: Remove extra lines

**Review by dayanandviveki — APPROVED** [review]
Need small small changes

File: prashant/HTML/Assignment7/Login.html (hunk 2 above)
Comment (Pawan1005) [inline prashant/HTML/Assignment7/Login.html:78]: make proper indentation

File: prashant/HTML/Assignment7/homepage.css [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+.mx-background-top-linear {
+    background: linear-gradient(45deg, blue 48%, #1b1e21 48%);   
+}
+
+.nav-link {
+    margin-left: 10px;
+    padding-left: 10px;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment7/homepage.css:49]: remove extra lines

File: prashant/HTML/Assignment7/login.css [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+.mx-background-top-linear {
+    background: linear-gradient(45deg, blue 48%, #1b1e21 48%);    
+}
+
+.nav-link {
+    margin-left: 10px;
+    padding-left: 10px;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment7/login.css:118]: remove extra lines

File: prashant/HTML/Assignment7/personaldetail.css [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+
+.mx-background-top-linear {
+    background: linear-gradient(45deg, blue 48%, #1b1e21 48%);
+    
+}
+
+.nav-link {
+    margin-left: 10px;
+    padding-left: 10px;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment7/personaldetail.css:141]: remove extra lines

File: prashant/HTML/Assignment7/personaldetail.js [hunk 14]
```diff
@@ -0,0 +1,5 @@
+$(document).ready(function(){
+    $('#birth-date').mask('00/00/0000');
+    $('#phone-number').mask('0000-0000');
+   })
+
```
Comment (Pawan1005) [inline prashant/HTML/Assignment7/personaldetail.js:6]: add new at the end of this file

File: prashant/HTML/Assignment7/professional.css [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+.mx-background-top-linear {
+    background: linear-gradient(45deg, blue 48%, #1b1e21 48%);
+    
+}
+.nav-link {
+    margin-left: 10px;
+    padding-left: 10px;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment7/professional.css:141]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: prashant/HTML/Assignment7/homepage.css (hunk 11 above)
Comment (Prashant698) [reply prashant/HTML/Assignment7/homepage.css:49]: done sir

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**